src/ConfigHelper.ts

```diff
@@ -121,7 +121,18 @@ let NATIVE_THREAD_PROPS_WHITELIST: Record<string, boolean> = {
   placeholderTextColor: true,
 };
 
+function assertNoOverlapInLists() {
+  for (const key in NATIVE_THREAD_PROPS_WHITELIST) {
+    if (key in UI_THREAD_PROPS_WHITELIST) {
+      throw new Error(
+        `[Reanimated] Property \`${key}\` was whitelisted both as UI and native prop. Please remove it from one of the lists.`
+      );
+    }
+  }
+}
+
 function configureProps(): void {
+  assertNoOverlapInLists();
```

Comment:
I think this check should be at the end of function, to check overlapping after modification of both set.

Reply:
yes

Reply:
Actually it is 😅
Sorry, my bad.

Reply:
:swiniok: